### PR TITLE
Add russian localization for Java 11 monitor

### PR DESCRIPTION
**Pull request by bzzitsme**
Screenshot 2022-03-11 at 15 59 45

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@Pldi23

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Recommended_Java_Version=\u0420\u0435\u043A\u043E\u043C\u0435\u043D\u0434\u0443\u0435\u043C\u0430\u044F \u0432\u0435\u0440\u0441\u0438\u044F \u0434\u043B\u044F \u0437\u0430\u043F\u0443\u0441\u043A\u0430 Jenkins - Java 11. \u0420\u0430\u0441\u0441\u043C\u043E\u0442\u0440\u0438\u0442\u0435 \u0432\u043E\u0437\u043C\u043E\u0436\u043D\u043E\u0441\u0442\u044C \u043E\u0431\u043D\u043E\u0432\u043B\u0435\u043D\u0438\u044F.
+More\ Info=\u0411\u043E\u043B\u044C\u0448\u0435 \u0438\u043D\u0444\u043E\u0440\u043C\u0430\u0446\u0438\u0438
```
Comment (Pldi23) [inline core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties:2]: for me it's more native to translate like that: `Подробнее`
For example google use the same pattern:
Screenshot 2022-03-11 at 17 30 12

**Review by Pldi23 — COMMENTED** [review]
Welcome to Jenkins community!

File: core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties (hunk 1 above)
Comment (bzzitsme) [reply core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties]: Changed it to `Подробнее`
Screenshot 2022-03-14 at 10 16 40

**Review by Pldi23 — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**